extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -232,8 +233,10 @@ public function execute() {
 
 		// Override wikia.com related config early when requesting a fandom.com wiki
 		if ( strpos( $this->mServerName, '.' . $wgFandomBaseDomain ) !== false ) {
-			$GLOBALS['wgServicesExternalDomain'] = "https://services.{$wgFandomBaseDomain}/";
-			$GLOBALS['wgCookieDomain'] = ".{$wgFandomBaseDomain}";
+			$domain = $wgDevelEnvironment ? $wgFandomDevDomain : $wgFandomBaseDomain;
```

Comment:
Doesn't `$wgFandomDevDomain` contain the devbox name like `mech.fandom-dev.pl`? If so, the `wgServicesExternalDomain` will become `https://services.mech.fandom-dev.pl/` which is wrong.

Reply:
fix in 6f3b20d2297